app/main.py

```diff
@@ -1,8 +1,34 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(
+            self,
+            name: str,
+            age: int
+    ) -> None:
+        self.name = name
+        self.age = age
 
```

Comment:
The task says: This class also should have a class attribute people, it is a dict that stores Person instances by their name. The __init__ method should add elements to this attribute.

You are not adding elements into people in `__init__`